integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -115,12 +115,14 @@ class RegionForm(CustomModelForm):
         required=False,
     )
 
-    deepl_midyear_start_enabled = forms.BooleanField(
+    mt_midyear_start_enabled = forms.BooleanField(
         required=False,
-        label=_("DeepL budget year start differs from renewal date"),
+        label=_(
+            "Budget year start for foreign language translation differs from renewal date"
```

Comment:
```suggestion
            "Budget year start differs from the renewal date"
```

I think we could make this a bit shorter, because the whole section is called "Foreign language machine translation" and it should be clear, that this field is also about foreign lang translation.
Otherwise it doesn't look nice on the form:
![image](https://github.com/digitalfabrik/integreat-cms/assets/115008338/73aacc9f-0532-45a0-b6e6-3790401eab67)

Or alternatively the html style can be adjusted.

Reply:
I thought it may be confusing for users which MT providers are covered by this budget (whether SAMM.I included or not).

But SAMM.AI has its own block so I think it's ok to make the sentence short.